src/reanimated2/core.ts

```diff
@@ -213,6 +213,22 @@ if (!isWeb() && isConfigured()) {
       info: runOnJS(capturableConsole.info),
     };
     _setGlobalConsole(console);
+
+    const nativeRequestAnimationFrame = global.requestAnimationFrame;
+    let callbacks: Array<(time: number) => void> = [];
+    global.requestAnimationFrame = (
+      callback: (time: number) => void
+    ): number => {
+      callbacks.push(callback);
+      if (callbacks.length === 1) {
+        nativeRequestAnimationFrame((timestamp) => {
+          const currentCallbacks = callbacks;
+          callbacks = [];
+          currentCallbacks.forEach((f) => f(timestamp));
+        });
+      }
+      return -1;
```

Comment:
Can we add a comment here that explains that technically `requestAnimationFrame` should return `requestID` (number) that can be passed to `cancelAnimationFrame` to cancel the callback but apparently we neither use the output of `requestAnimationFrame` throughout our codebase nor implement `cancelAnimationFrame`?